File: origin/BasketMerge/Basket/src/com/example/basket/Product.java
package com.example.basket;

import org.codehaus.jackson.annotate.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Product 
{
	int ISBN;
	public int getISBN() {
		return ISBN;
	}
	public void setISBN(int iSBN) {
		ISBN = iSBN;
	}
	public int getWidth() {
		return width;
	}
	public void setWidth(int width) {
		this.width = width;
	}
	public int getHeight() {
		return height;
	}
	public void setHeight(int height) {
		this.height = height;
	}
	public int getDepth() {
		return depth;
	}
	public void setDepth(int depth) {
		this.depth = depth;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public String getFeatures() {
		return features;
	}
	public void setFeatures(String features) {
		this.features = features;
	}
	int width;
	int height;
	int depth;
	String name;
	String description;
	String features;
	
	
}

File: Basket.js/product.js
/**
 * New node file
 */
module.exports =  { 
	product : function (name, ISBN,Manufacturer,width,height,depth,description,features)
	{
		
		this.name = name;
		this.ISBN = ISBN;
		this.Manufacturer =  Manufacturer;
		this.width=width;
		this.height=height;
		this.depth=depth;
		this.description=description;
		this.features=features;
		
		
	}
};
